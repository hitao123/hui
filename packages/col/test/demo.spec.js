describe('Col component', () => {
  it('should show Col', () => {

  });
});
