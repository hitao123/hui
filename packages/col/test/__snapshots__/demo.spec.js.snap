// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders demo correctly 1`] = `
<div>
  <div>
    <div class="h-row">
      <div class="h-col h-col--8">span: 8</div>
      <div class="h-col h-col--8">span: 8</div>
      <div class="h-col h-col--8">span: 8</div>
    </div>
    <div class="h-row">
      <div class="h-col h-col--4">span: 4</div>
      <div class="h-col h-col--10 h-col--offset-4">offset: 4, span: 10</div>
    </div>
    <div class="h-row">
      <div class="h-col h-col--12 h-col--offset-12">offset: 12, span: 12</div>
    </div>
  </div>
  <div>
    <div class="h-row" style="margin-left:-10px;margin-right:-10px;">
      <div class="h-col h-col--8" style="padding-left:10px;padding-right:10px;">span 8</div>
      <div class="h-col h-col--8" style="padding-left:10px;padding-right:10px;">span 8</div>
      <div class="h-col h-col--8" style="padding-left:10px;padding-right:10px;">span 8</div>
    </div>
  </div>
  <div>
    <div class="h-row h-row--flex">
      <div class="h-col h-col--6">span: 6</div>
      <div class="h-col h-col--6">span: 6</div>
      <div class="h-col h-col--6">span: 6</div>
    </div>
    <div class="h-row h-row--flex h-row--justify-center">
      <div class="h-col h-col--6">span: 6</div>
      <div class="h-col h-col--6">span: 6</div>
      <div class="h-col h-col--6">span: 6</div>
    </div>
    <div class="h-row h-row--flex h-row--justify-end">
      <div class="h-col h-col--6">span: 6</div>
      <div class="h-col h-col--6">span: 6</div>
      <div class="h-col h-col--6">span: 6</div>
    </div>
    <div class="h-row h-row--flex h-row--justify-space-between">
      <div class="h-col h-col--6">span: 6</div>
      <div class="h-col h-col--6">span: 6</div>
      <div class="h-col h-col--6">span: 6</div>
    </div>
    <div class="h-row h-row--flex h-row--justify-space-around">
      <div class="h-col h-col--6">span: 6</div>
      <div class="h-col h-col--6">span: 6</div>
      <div class="h-col h-col--6">span: 6</div>
    </div>
  </div>
</div>
`;
